sdl-core/src/main/scala/io/smartdatalake/workflow/connection/JdbcTableConnection.scala

```diff
@@ -149,68 +164,27 @@ private[smartdatalake] object SQLCatalog {
  */
 private[smartdatalake] class DefaultSQLCatalog(connection: JdbcTableConnection) extends SQLCatalog(connection) {
   override def isDbExisting(db: String)(implicit session: SparkSession): Boolean = {
-    val cntTableInCatalog =
-      if (Environment.enableJdbcCaseSensitivity)
-        s"select count(*) from INFORMATION_SCHEMA.SCHEMATA where TABLE_SCHEMA='$db'"
-      else
-        s"select count(*) from INFORMATION_SCHEMA.SCHEMATA where upper(TABLE_SCHEMA)=upper('$db')"
+    val cntTableInCatalog = s"select count(*) from INFORMATION_SCHEMA.SCHEMATA where TABLE_SCHEMA=$db"
```

Comment:
What if <db> is a case sensitive name? Here we need to implement a more specific query, as if the <db> is configured with quotes in Hocon, it will never be found in a catalog. We would need to check if <db> is quoted, if yes remove the quotes and make a case sensitive comparision, otherwise we make a case insensitive comparision (upper both sides).
What do you think?

Reply:
You're absolutely right of course, just passing the quoted string to the database doesn't work in case of isDbExisting. 
As discussed, fixed in https://github.com/smart-data-lake/smart-data-lake/pull/328/commits/49f04537424e96689962403388d70ffc04bf3da2.